components/director/internal/domain/apptemplate/service.go

```diff
@@ -174,7 +167,7 @@ func (s *service) ListLabels(ctx context.Context, appTemplateID string) (map[str
 		return nil, fmt.Errorf("application template with ID %s doesn't exist", appTemplateID)
 	}
 
-	labels, err := s.labelRepo.ListForObject(ctx, appTenant, model.AppTemplateLabelableObject, appTemplateID)
+	labels, err := s.labelRepo.ListForObject(ctx, "", model.AppTemplateLabelableObject, appTemplateID) // tenent is not needed for AppTemplateLabelableObject
```

Comment:
Only a suggestion - we can replicate this func as ListGlobalForObject and we won't be providing a tenant for it.